objects/d3fend.json

```
@@ -1,19 +1,20 @@
 {
     "caption": "MITRE D3FEND™",
     "name": "d3fend",
-    "description": "The <a target='_blank' href='https://d3fend.mitre.org'>MITRE D3FEND™</a> object describes the tactic, technique & sub-technique associated with a countermeasure as defined in <a target='_blank' href='https://d3fend.mitre.org/'>DEFEND Matrix<sup>TM</sup></a>.",
+    "description": "The MITRE D3FEND™ object describes the tactic, technique & sub-technique associated with a countermeasure.",
+    "references": [{"url": "https://d3fend.mitre.org", "description": "D3FEND™ Matrix"}],
     "extends": "object",
     "attributes": {
       "d3f_tactic": {
-        "description": "The Tactic object describes the tactic ID and/or name that is associated with a countermeasure, as defined by <a target='_blank' href='https://d3fend.mitre.org'>D3FEND Matrix<sup>TM</sup></a>.",
+        "description": "The Tactic object describes the tactic ID and/or name that is associated with a countermeasure.",
         "requirement": "recommended"
       },
       "d3f_technique": {
-        "description": "The Defend Technique object describes the technique ID and/or name associated with a countermeasure, as defined by <a target='_blank' href='https://d3fend.mitre.org'>D3FEND Matrix<sup>TM</sup></a>.",
+        "description": "The Technique object describes the technique ID and/or name associated with a countermeasure.",
         "requirement": "recommended"
       },
       "version": {
-        "description": "The <a target='_blank' href='https://d3fend.mitre.org'>D3FEND Matrix<sup>TM</sup></a> version.",
+        "description": "The D3FEND<sup>TM</sup> Matrix version.",
```

```suggestion
        "description": "The D3FEND™ Matrix version.",
```
Change to TM character instead of HTML version (for consistency).